dih_models/parameters.py

```
@@ -14433,9 +14433,389 @@ def calculate_personal_lifetime_wealth_conservative_baseline(
+    latex_symbol=r"s_{queue}",
+)
+
+STATE_RTT_QUEUE_CLEARANCE_YEARS = Parameter(
+    float(DISEASES_WITHOUT_EFFECTIVE_TREATMENT)
+    / (float(NEW_DISEASE_FIRST_TREATMENTS_PER_YEAR) + float(STATE_RTT_PROVEN_TREATMENTS_ANNUAL) * float(STATE_RTT_EVALUATIONS_QUEUE_SHARE)),
+    manual_ref="knowledge/appendix/state-right-to-trial-impact.qmd",
+    source_type="calculated",
+    description="Years to find first treatments for all currently untreated diseases with the 50-state registry running and no public subsidies. The registry's queue-directed discoveries add to the status quo worldwide rate of ~15 first treatments per year. Same formula structure as the status quo (443 years) and dFDA full-deployment (36 years) estimates.",
+    display_name="Untreated Disease Queue Clearance (50-State Registry, Unsubsidized)",
+    unit="years",
+    formula="DISEASES_WITHOUT_EFFECTIVE_TREATMENT ÷ (NEW_DISEASE_FIRST_TREATMENTS_PER_YEAR + STATE_RTT_PROVEN_TREATMENTS_ANNUAL × STATE_RTT_EVALUATIONS_QUEUE_SHARE)",
```

Ruff (0.16.1)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)